ansible/roles/optimizations/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Disable daily upgrades
+  systemd:
+    name: "{{ item }}"
+    state: disabled
+    daemon-reload: yes
+  with_items:
+    - 'apt-daily-upgrade.timer'
+    - 'apt-daily.timer'
+    - 'apt-daily.service'
```

Comment:
You need according to the answer of this [problem](https://superuser.com/questions/1327884/how-to-disable-daily-upgrade-and-clean-on-ubuntu-16-04), you need to remove `unattended-upgrades` , stopped `apt-daily.timer` service before disable these services.
The code might be like this : 
```yaml
---

- name: Remove unattended-upgrades package
  ansible.builtin.apt:
    name: unattended-upgrades
    state: absent

- name: stopped daily upgrades
  systemd:
    name: "apt-daily.timer"
    state: stopped

- name: Disable daily upgrades
  systemd:
    name: "{{ item }}"
    state: disabled
    daemon-reload: yes
  with_items:
    - 'apt-daily.timer'
    - 'apt-daily.service'
```